suite/tests/CMakeLists.txt

```diff
@@ -1386,6 +1386,14 @@ function(torun test key source native standalone_dr dr_ops exe_ops added_out pas
       string(REGEX REPLACE ";" "@" nudge_arg "${nudge_arg}")
       # clear client from dr command and run native
       string(REGEX REPLACE ";-quiet;(.*);--" ";-no_inject;--;" rundr "${rundr}")
+    elseif ("${runall}" MATCHES "<detach>")
+      # move params to nudge
```

Comment:
style nit: We prefer capitalized sentences with end punctuation for all new comments: so "Move params to nudge.".  Ditto below.

Reply:
I have updated it